var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -0,0 +1,164 @@
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+# Author: Matteo Giantomassi <matteo.giantomassiNOSPAM AT uclouvain.be>
+# Date: October 11, 2016
+
+from spack import *
+
+
+class Abinit(Package):
+    """ABINIT is a package whose main program allows one to find the total energy,
+    charge density and electronic structure of systems made of electrons and nuclei
+    (molecules and periodic solids) within Density Functional Theory (DFT), using
+    pseudopotentials and a planewave or wavelet basis. ABINIT also includes options
+    to optimize the geometry according to the DFT forces and stresses, or to perform
+    molecular dynamics simulations using these forces, or to generate dynamical
+    matrices, Born effective charges, and dielectric tensors, based on
+    Density-Functional Perturbation Theory, and many more properties. Excited states
+    can be computed within the Many-Body Perturbation Theory (the GW approximation
+    and the Bethe-Salpeter equation), and Time-Dependent Density Functional Theory
+    (for molecules). In addition to the main ABINIT code, different utility programs
+    are provided."""
+
+    homepage = "http://www.abinit.org"
+    url      = "http://ftp.abinit.org/abinit-8.0.8b.tar.gz"
+
+    # Versions before 8.0.8b are not supported.
+    version("8.0.8b", "abc9e303bfa7f9f43f95598f87d84d5d")
+
+    variant('mpi', default=True, description='Builds with MPI support. Requires MPI2+')
+    variant('openmp', default=False, description='Enables OpenMP threads. Use threaded FFTW3')
+    variant('scalapack', default=False, description='Enables scalapack support. Requires MPI')
+    #variant('elpa', default=False, description='Uses elpa instead of scalapack. Requires MPI')
+
+    # TODO: To be tested. It was working before the last `git pull` but now all tests crash.
+    # For the time being, the default is netcdf3 and the internal fallbacks provided by Abinit.
+    variant('hdf5', default=False, description='Enables HDF5+Netcdf4 (parallel version). WARNING: experimental')
+
+    # Add dependencies
+    depends_on("blas", when="~openmp")
+    depends_on("blas+openmp", when="+openmp")
```

Comment:
currently you can't forward options to virtual packages, see https://github.com/LLNL/spack/issues/1712.

I would comment this `depends_on` and add `TODO` here or a small note.